we_get/core/module.py

```diff
@@ -20,22 +19,19 @@ def http_get_request(self, url):
         """ http_request: create HTTP request.
           @return: data.
         """
-        opener = urllib.request.build_opener()
-        opener.addheaders = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
-        try:
-            res = opener.open(url).read().decode()
-            return res
-        except URLError as err:
+        headers = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
+        res = requests.get(url, headers)
+        if res.status_code == 200:
+            return res.text
+        else: 
             print('Error when opening following url.\n{}'.format(url))
             raise err
 
     def http_custom_get_request(self, url, headers):
         """ http_custom_get_request: HTTP GET request with custom headers.
           @return: data.
         """
-        opener = urllib.request.build_opener()
-        opener.addheaders = headers
-        return opener.open(url).read()
+        return requests.get(url, headers).text
```

Comment:
that code mean headers is given to `params` parameter,

https://docs.python-requests.org/en/latest/api/#requests.get

should be this `requests.get(url, headers=headers).text`